front/app/containers/Admin/dashboard/reports/PollReport.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { memo } from 'react';
+import { adopt } from 'react-adopt';
+import { InjectedIntlProps } from 'react-intl';
+import GetPollQuestions, {
+  GetPollQuestionsChildProps,
+} from 'resources/GetPollQuestions';
+import { isNilOrError } from 'utils/helperUtils';
+import {
+  GraphCard,
+  GraphCardInner,
+  GraphsContainer,
+} from 'containers/Admin/dashboard';
+import QuestionReport from './QuestionReport';
+import { SubSectionTitle } from 'components/admin/Section';
+import { injectIntl } from 'utils/cl-intl';
+
+interface InputProps {
+  participationContextId: string;
+  participationContextType: 'phase' | 'project';
+  participationContextTitle: string;
+}
+interface DataProps {
+  pollQuestions: GetPollQuestionsChildProps;
+}
+
+interface Props extends InputProps, DataProps {}
+
+const PollReport = memo(
+  ({
+    participationContextId,
+    participationContextType,
+    pollQuestions,
+    participationContextTitle,
+  }: Props & InjectedIntlProps) => {
+    return (
+      <div>
+        {!isNilOrError(participationContextTitle) &&
+          participationContextType === 'phase' && (
+            <SubSectionTitle>{participationContextTitle}</SubSectionTitle>
+          )}
+        <GraphsContainer>
+          {!isNilOrError(pollQuestions) &&
+            pollQuestions.length &&
```

Comment:
Personally not a fan of code like this. It works, but it hides what we're doing. In theory, this would also pass if length were -1 or [] or 'abc'. I think we should not always go for the shortest code possible, but clearly indicate what we're doing, so would make this pollQuestions.length > 0.